gateway/webhook/webhook.go

```diff
@@ -122,6 +125,53 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var postFrom url.Values
+	var multipartForm *multipart.Form
+
+	if r.Method == "GET" {
+		return
+	}
+	contentType := r.Header.Get("Content-Type")
+	if contentType == "application/x-www-form-urlencoded" {
+		if err := r.ParseForm(); err != nil {
+			http.Error(w, "Could not parse form", 400)
```

Comment:
can we use the `failRequest` to return error here too? Also increment the metrics count etc like above ```webhook.failRequest(w, r, response.GetStatus(response.NoWriteKeyInQueryParams), response.GetStatusCode(response.NoWriteKeyInQueryParams), "noWriteKey")
atomic.AddUint64(&webhook.ackCount, 1)```